### ISSUE TITLE
Smart(Dumb)watch support

### ISSUE THREAD

**Issue by dderby** [maemo-leste/bugtracker#318]
This is **not** a proposal to port Maemo Leste to smartwatches.

I used the [Fossil MetaWatch](https://en.wikipedia.org/wiki/MetaWatch) with javispedro's SmartOpenWatch[[1](https://git.javispedro.com/cgit/sowatch.git/)][[2](https://web.archive.org/web/20120217103005/http://forum.meego.com/showthread.php?t=4492)][[3](https://web.archive.org/web/20140804043625/https://gitorious.org/sowatch/pages/Home)] with Harmattan for about a year and had a great experience with it before I switched back to using the N900 as my main phone.  Unlike many modern smartwatches which can run independently of smartphones, this watch is more akin to a dumb terminal than a smartphone (IMO).  The entire functionality is driven from the smartphone's OS and its apps/applets run on the smartphone with the watch only needing to provide a display for them, plus input via a couples of buttons.  No software is installed on the watch other than the stock FreeRTOS firmware.

It would be great to get SmartOpenWatch (or something similar) running under Leste.  I don't think it would be too difficult to get something usable up and running.

**Comment by IMbackK** [comment]
we can close this as of topic

**Comment by MerlijnWajer** [comment]
I think technically it's a request for an application, no?

**Comment by dderby** [comment]
Indeed, perhaps more of a daemon than an application, but feel free to close this if it's only me that's interested in it.
